### PR TITLE
Web/home/firefox-and-svg-fix

### LINKED ISSUE #103: for photos, use `png` and compress them instead of `svg`
`olivia.svg` for e.g. is > 11MB (yikes) so use `png` and compress

promising suspect for #102

### LINKED ISSUE #109: Stories section text layout issue on FireFox
I stumbled upon this on FireFox when the screen width is below 900px.

<details>
  <summary>Text is far below the image</summary>
  
</details>

<details>
  <summary>Text is underneath the image</summary>
  
</details>
<br/>
Looks fine on Chrome.
<br/><br/>
EDIT: Hover also doesn't work. :(

### PR DESCRIPTION
**Pull request by chamkank**
:tickets: **Ticket(s)**: Closes #109, closes #103 

---

## :construction_worker: Changes

1. Fixes the FireFox issue with text layout & hover behavior in #109  
2. Also added vendor prefixes for `display: flex`
3. svg -> png for photos (#103)
## :flashlight: Testing Instructions

Stories should look and work fine in FireFox when screen width < 900px.

### CODE REVIEW

File: web/styles/utils.sass [hunk 1]
```diff
@@ -135,8 +139,7 @@
 .hover-show
   opacity: 0
   -webkit-transition: $transition-time
-  @media (hover:hover)
```
Comment (mingyokim) [inline web/styles/utils.sass]: why are we removing this line?

File: web/components/home/Home.js [hunk 2]
```diff
@@ -233,7 +233,7 @@ class Home extends React.Component {
                       <div className="pad-top-l pad-bottom-xxl pad-sides-m hover-description flex dir-col jc-center fill-height">{userToText[person]}</div>
                     </div>
                   </div>
-                  <div className="scale-show-full-screen pad-top-m pad-bottom-xxl flex dir-col jc-center fill-height">{userToText[person]}</div>
+                  <div className="stories-text scale-show-full-screen pad-top-m pad-bottom-xxl flex dir-col jc-center">{userToText[person]}</div>
```
Comment (mingyokim) [inline web/components/home/Home.js]: is `stories-text` an existing class? I don't see this class in this diff

File: web/styles/utils.sass (hunk 1 above)
Comment (chamkank) [reply web/styles/utils.sass]: It seems to break the hover in Firefox

File: web/components/home/Home.js (hunk 2 above)
Comment (chamkank) [reply web/components/home/Home.js]: Oops I created that while debugging, I'll remove it

**Pull request merged**